src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs

```diff
@@ -12,55 +12,103 @@
 using Uno.UI.RemoteControl.Messages;
 using System.Runtime.Loader;
 using Microsoft.Extensions.Configuration;
+using System.Reflection;
+using System.Runtime.Remoting.Messaging;
 
 namespace Uno.UI.RemoteControl.Host
 {
 	internal class RemoteControlServer : IRemoteControlServer, IDisposable
 	{
-		private readonly Dictionary<string, IServerProcessor> _processors = new Dictionary<string, IServerProcessor>();
+		private readonly Dictionary<string, IServerProcessor> _processors = new();
 
 		private System.Reflection.Assembly? _loopAssembly;
 		private WebSocket? _socket;
+		private string? _appInstanceId;
 		private readonly IConfiguration _configuration;
-		private readonly AssemblyLoadContext _loadContext;
+		private readonly static Dictionary<string, AssemblyLoadContext> _loadContexts = new();
```

Comment:
Can we move this to the top of the class to separate it away from the instance fields?